web/concrete/config/services.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return array(
+    'apache' => array('Concrete\Core\Service\HTTP\Apache'),
+    'nginx' => 'Concrete\Core\Service\HTTP\Nginx',
```

Comment:
Historical, I think. This pull request has a looooong story behind it...
Fixed in https://github.com/mlocati/concrete5/commit/31e44bfeef1594c9a0f0b078fc16ea5fb5784af9